perf/benchmark/run_benchmark_job.sh

```diff
@@ -111,6 +111,45 @@ function setup_fortio_and_prometheus() {
     export FORTIO_SERVER_POD
 }
 
+function port_forward_envoy() {
+  kubectl -n "${NAMESPACE}" port-forward "${1}" 15000 &
+}
+
+function kill_envoy_port() {
+    kill "$(lsof -t -i:15000)"
+}
+
+function collect_config_dump() {
+  kill_envoy_port
+  port_forward_envoy "${FORTIO_CLIENT_POD}"
```

Comment:
might be easier to just `kubectl exec`?

Reply:
@carolynhu I agree kubectl exec does not involve any races or waiting like port forward.

Reply:
updated to use kubectl exec